src/server/metadata/cmds/cmds.go

```diff
@@ -18,15 +21,21 @@ func Cmds(mainCtx context.Context, pachctlCfg *pachctl.Config) []*cobra.Command
 	var commands []*cobra.Command
 
 	editMetadata := &cobra.Command{
+		Use:   `{{alias}} [<object type: project> <object picker> <operation: add|edit|delete|set> <data: key=value, key, '{"key":"value","key2":"value2"}'>]...`,
 		Short: "Edits an object's metadata",
 		Long:  "Edits an object's metadata",
```

Comment:
I think that our long documentation is typically in the form of a sentence ending in a period.

Reply:
Done.